### PR TITLE
docs: Update READMEs

### PR DESCRIPTION
**Pull request by mathio**
Is this documentation sufficient? Have we outgrown the README file and should generate a Github Pages docs with more examples?

### CODE REVIEW

**Review by antoniofull — APPROVED** [review]

**Comment by antoniofull** [comment]
> Is this documentation sufficient? Have we outgrown the README file and should generate a Github Pages docs with more examples?

@mathio  I would go with a simple GH page

File: packages/embed/README.md [hunk 1]
```diff
@@ -2,7 +2,108 @@
...
+- to pass `string[]` use coma-separated string, eg. `transitiveSearchParams: ['foo', 'bar']` becomes `data-tf-transitive-search-params="foo,bar"`
+- to pass `object` pass coma-separated key=value pairs, eg. `hidden: { foo: "f", bar: "b" }` becomse `data-tf-hidden="foo=f,bar=b"`
+
+You can find examples for specific use-cases in our demos:
+
+- [HTML demo](./packages/demo-html)
+- [Webpack demo](./packages/demo-webpack)
+- [React demo](./packages/demo-react)
+- [Next.js demo](./packages/demo-nextjs)
+
```
Comment (kb-typeform) [inline packages/embed/README.md:85]: I'd check the links above work

File: packages/embed/README.md [hunk 2]
```diff
@@ -2,7 +2,108 @@
...
+| open                   | string      | open embed based on user action (see below)                                                                                | `undefined`                                                   |
+| openValue              | number      | based on `open` (see below)                                                                                                | `undefined`                                                   |
+| onReady                | function    | fires when the form is loaded                                                                                              | `undefined`                                                   |
+| onSubmit               | function    | fires when user submits the form                                                                                           | `undefined`                                                   |
+| onQuestionChanged      | function    | fires when user navigates between form questions                                                                           | `undefined`                                                   |
+
```
Comment (kb-typeform) [inline packages/embed/README.md:72]: 👍  nice props table

File: packages/embed/README.md [hunk 3]
```diff
@@ -2,7 +2,108 @@
...
+- widget: `createWidget('<form-id>', options)` or `data-tf-widget="<form-id>>"`
+- popup: `createPopup('<form-id>', options)` or `data-tf-popup="<form-id>>"`
+- slider: `createSlider('<form-id>', options)` or `data-tf-slider="<form-id>>"`
+- sidetab: `createSidetab('<form-id>', options)` or `data-tf-sidetab="<form-id>>"`
+- popover: `createPopover('<form-id>', options)` or `data-tf-popover="<form-id>>"`
+
+`form-id` is string, you can find it in your typeform URL `https://form.typeform.com/to/<form-id>`
+
+`options` is an object with optional properties:
+
```
Comment (kb-typeform) [inline packages/embed/README.md:54]: maybe add title 'Props' here?

File: packages/embed/README.md [hunk 4]
```diff
@@ -2,7 +2,108 @@
...
+You can find examples for specific use-cases in our demos:
+
+- [HTML demo](./packages/demo-html)
+- [Webpack demo](./packages/demo-webpack)
+- [React demo](./packages/demo-react)
+- [Next.js demo](./packages/demo-nextjs)
+
+### Custom Launch Options
+
+Properties `open` and `openValue` apply only to embed types that are opened by user action (all except widget). They defined when to automatically open the typeform.
```
Comment (kb-typeform) [inline packages/embed/README.md:88]: typo: "They defined" => "They define"

File: packages/embed/README.md [hunk 5]
```diff
@@ -2,7 +2,108 @@
...
+- as user scrolls the page
+  - `open: 'scroll'`
+  - `openValue` percentage of page scrolled (0 - 100) to open the form
+- after time elapsed
+  - `open: 'time'`
+  - `openValue` number of seconds to wait before opening the form
+
+For details see [behavioral demo](../demo-html/public/behavioral-html).
+
+## Development and Contribution
```
Comment (kb-typeform) [inline packages/embed/README.md:107]: I think the 'contribution' part could be made clearer

**Review by mrisoli — APPROVED** [review]

File: packages/demo-webpack/README.md [hunk 6]
```diff
@@ -8,4 +8,4 @@ import { createWidget } from "@typeform/embed";
 createWidget("<form id>", { container: document.querySelector("#container") });
 ```
 
-Use `yarn start` to run the demo in browser: http://localhost:8080
+Run `yarn build` in [embed package](../embed). Then run `yarn start` to run the demo in browser: http://localhost:8080
```
Comment (picsoung) [inline packages/demo-webpack/README.md:11]: Add section title `Example`
And maybe instructions to clone this specific part of the repo, to just have this example?

Anything about common issues with SSR and webpack?
Should we add a section about how to handle this?

File: packages/embed/README.md [hunk 7]
```diff
@@ -2,7 +2,108 @@
...
+Install:
+
+```shell
+yarn add @typeform/embed@next
+```
+
+Import the lib, CSS and create your embed:
+
+```javascript
+import { createWidget } from '@typeform/embed@next'
```
Comment (picsoung) [inline packages/embed/README.md:18]: provide complete import?
`{ createWidget, createPopup, createSidebar, ..}`

**Review by picsoung — CHANGES_REQUESTED** [review]
Hey :)

good to see some progress made in the embed SDK!
And great to provide examples for each framework.

Maybe it could be interesting to leverage GitHub wiki to organize documentation.

My general comments are probably more about the overall organization of the sections, and explicitly documenting everything for our end users.

Let me know if it does not make sense :)

File: README.md [hunk 8]
```diff
@@ -8,9 +8,9 @@ Install all dependencies from repository root:
 yarn
 ```
 
-Libraries:
+For usage see README of each library:
```
[the remark this answers is not in the document]
Comment (mathio) [reply README.md:11]: Each lib (in packages/ dir) will have its own docs.

File: packages/demo-webpack/README.md (hunk 6 above)
Comment (mathio) [reply packages/demo-webpack/README.md:11]: SSR will "just work", no workaround necessary.

File: packages/embed/README.md (hunk 7 above)
Comment (mathio) [reply packages/embed/README.md:18]: There might be more to come in future. This is supposed to be just a quick and short example. Full list is in `Embed types` below.

File: packages/embed/README.md [hunk 9]
```diff
@@ -2,7 +2,108 @@
...
+import '@typeform/embed/build/css/widget.css'
```
[the remark this answers is not in the document]
Comment (mathio) [reply packages/embed/README.md:19]: Good point. I dont know if we are at a point where we can support custom CSS. Lets leave it as is for now.

File: packages/embed/README.md (hunk 2 above)
Comment (mathio) [reply packages/embed/README.md:72]: The table would be messy with examples. I want to deploy examples somewhere, but that is yet to be determined.

**Review by prilutskiy — APPROVED** [review]

File: packages/embed/README.md [hunk 10]
```diff
@@ -2,7 +2,108 @@
...
+| open                   | string      | open embed based on user action (see below)                                                                                | `undefined`                                                   |
+| openValue              | number      | based on `open` (see below)                                                                                                | `undefined`                                                   |
+| onReady                | function    | fires when the form is loaded                                                                                              | `undefined`                                                   |
+| onSubmit               | function    | fires when user submits the form                                                                                           | `undefined`                                                   |
+| onQuestionChanged      | function    | fires when user navigates between form questions                                                                           | `undefined`                                                   |
```
[the remark this answers is not in the document]
Comment (mathio) [reply packages/embed/README.md:86]: I agree and I need to investigate this to make sure its correct information. I will add that later.

**Pull request merged**